src/instructlab/model/evaluate.py

```diff
@@ -286,71 +350,59 @@ def evaluate(
     )
 
     if benchmark == "mt_bench":
-        # TODO: Replace temp Popen hack with serving library calls. Current library doesn't support server-model-name.
         print("Generating answers...")
-        proc = None
+        server = None
         try:
-            proc = subprocess.Popen(
-                [
-                    "python",
-                    "-m",
-                    "vllm.entrypoints.openai.api_server",
-                    "--model",
-                    model,
-                    "--tensor-parallel-size",
-                    "1",
-                    "--served-model-name",
-                    "test_model",
-                ]
+            server, api_base = launch_server(
+                ctx,
+                model,
+                "test_model",
+                max_workers,
+                tls_insecure,
+                tls_client_cert,
+                tls_client_key,
+                tls_client_passwd,
             )
-            time.sleep(60)
-            evaluator.gen_answers("http://localhost:8000/v1")
+            evaluator.gen_answers(api_base)
         finally:
-            if proc:
-                proc.terminate()
+            if server is not None:
+                server.shutdown()
 
         print("Evaluating answers...")
-        proc = None
         try:
-            proc = subprocess.Popen(
-                [
-                    "python",
-                    "-m",
-                    "vllm.entrypoints.openai.api_server",
-                    "--model",
-                    judge_model,
-                    "--tensor-parallel-size",
-                    "1",
-                    "--served-model-name",
-                    "judge_model",
-                ]
-            )
-            time.sleep(60)
-            overall_score, qa_pairs, turn_scores = evaluator.judge_answers(
-                "http://localhost:8000/v1"
+            server, api_base = launch_server(
+                ctx,
+                model,
+                "judge_model",
```

Comment:
Should `judge_model` be a constant somewhere in `configuration.py`?

Reply:
I've moved it to a constant in evaluate.  I don't think it has any value outside of this scope.  But let me know if there is logic to move it higher.